src/renderers/dom/shared/__tests__/ReactDOMComponent-test.js

```diff
@@ -318,13 +317,33 @@ describe('ReactDOMComponent', () => {
       expect(container.firstChild.className).toEqual('');
     });
 
-    it('should properly update custom attributes on custom elements', () => {
+    it('should properly update custom properties on custom elements', () => {
```

Comment:
If this pull request is merged, then the only way to set attributes on a custom element would be with refs (`<custom-element ref={el => el.setAttribute('my-attr', val)} />`).